build.cake

```diff
@@ -796,72 +888,91 @@ string GetNuGetPackage(string packageId, string packageVersion)
     return filename;
 }
 
-void ExtractNuGetPackage (IPackage package, FrameworkName frameworkName, string tempContentPath, string destination) {
-    if (package != null) {
-        IEnumerable<IPackageAssemblyReference> assemblies;
-        VersionUtility.TryGetCompatibleItems (frameworkName, package.AssemblyReferences, out assemblies);
-
-        // Move assemblies.
-        foreach (var i in assemblies) {
-            if (!FileExists (destination + "/" + i.Name)) {
-                MoveFile (tempContentPath + "/" + i.Path, destination + "/" + i.Name);
-            }
-        }
-
+FilePathCollection ResolveDllFiles(string tempContentPath) 
+{
+    var uwpPath = $"{tempContentPath}/lib/{UwpFrameworkName.GetShortFolderName()}/*.dll";
+    var files = GetFiles(uwpPath);
+    if (files.Any()) 
+    {
+        return files;
+    }
+    var netStandard2Path = $"{tempContentPath}/lib/{Netstandard2FrameworkName.GetShortFolderName()}/*.dll";
+    files = GetFiles(netStandard2Path);
+    if (files.Any()) 
+    {
+        return files;
+    }
+    if (files.Any()) 
```

Comment:
?

Reply:
This is to check if enumerable is not empty.